cli/src/main/java/com/liferay/blade/cli/gradle/GradleTooling.java

```diff
@@ -69,27 +71,35 @@ public static boolean isLiferayModule(File buildDir) throws Exception {
 
 			ModelBuilder<T> modelBuilder = connection.model(modelClass);
 
-			Path tempPath = Files.createTempDirectory("blade-tooling-model");
+			Path tempPath = Files.createTempDirectory("tooling");
 
-			InputStream in = GradleTooling.class.getResourceAsStream("/deps.zip");
+			InputStream in = GradleTooling.class.getResourceAsStream("/tooling.zip");
 
 			FileUtil.unzip(in, tempPath.toFile());
 
-			String initScriptTemplate = FileUtil.collect(GradleTooling.class.getResourceAsStream("init.gradle"));
+			try (Stream<Path> toolingFiles = Files.list(tempPath)) {
+				String files = toolingFiles.map(
+					Path::toAbsolutePath
+				).map(
+					Path::toString
+				).map(
+					path -> "\"" + path + "\""
+				).collect(
+					Collectors.joining(", ")
+				);
 
-			String libsPath = tempPath.toString();
+				String initScriptTemplate = FileUtil.collect(GradleTooling.class.getResourceAsStream("init.gradle"));
 
-			libsPath = libsPath.replaceAll("\\\\", "/");
+				String initScriptContents = initScriptTemplate.replaceAll("%files%", files);
 
-			String initScriptContents = initScriptTemplate.replaceAll("%libsPath%", libsPath);
+				Path initPath = tempPath.resolve("init.gradle");
 
-			Path initPath = tempPath.resolve("init.gradle");
+				FileUtil.write(initScriptContents.getBytes(), initPath.toFile());
```

Comment:
Should we use `java.nio.file.Files.write(initPath, initScriptContents.getBytes())`?